### PR TITLE
Run all automation scripts except bootstrap using PowerShell 7

### PR DESCRIPTION
**Pull request by janegilring**
- Added PowerShell 7 installation to bootstrap.ps1
- Modified scheduled tasks to run pwsh.exe instead of powershell.exe

### CODE REVIEW

File: azure_jumpstart_arcbox/artifacts/Bootstrap.ps1 [hunk 1]
```diff
@@ -121,6 +121,20 @@ if ($flavor -eq "DataOps") {
 }
 
 # Installing tools
+
+Write-Header "Installing PowerShell 7"
```
Comment (fcabrera23) [inline azure_jumpstart_arcbox/artifacts/Bootstrap.ps1:125]: Checking official docs, the recommended installation is to go with Winget: https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.4#install-powershell-using-winget-recommended -> Any reason why not to use winget? Seems to be simpler

File: azure_jumpstart_arcbox/artifacts/Bootstrap.ps1 [hunk 2]
```diff
@@ -321,21 +335,21 @@ Write-Header "Configuring Logon Scripts"
 if ($flavor -eq "Full" -Or $flavor -eq "ITPro") {
     # Creating scheduled task for ArcServersLogonScript.ps1
     $Trigger = New-ScheduledTaskTrigger -AtLogOn
-    $Action = New-ScheduledTaskAction -Execute "PowerShell.exe" -Argument $Env:ArcBoxDir\ArcServersLogonScript.ps1
+    $Action = New-ScheduledTaskAction -Execute "pwsh.exe" -Argument $Env:ArcBoxDir\ArcServersLogonScript.ps1
```
Comment (fcabrera23) [inline azure_jumpstart_arcbox/artifacts/Bootstrap.ps1:338]: Maybe define pwsh variable and use it instead of pwsh.exe repeateadly

**Review by fcabrera23 — COMMENTED** [review]
Added a couple of comments

File: azure_jumpstart_arcbox/artifacts/Bootstrap.ps1 (hunk 1 above)
Comment (janegilring) [reply azure_jumpstart_arcbox/artifacts/Bootstrap.ps1:125]: Good question. The reason is that WinGet must be bootstrapped first, and the easiest way to do that on Server SKUs is to use `Repair-WinGetPackageManager.` That command will install both the WinGet CLI, but the gotcha is that the command only works on PowerShell 7 - not when run in Windows PowerShell. Hence, we get a catch 22 situation. For all other packages, we plan to leverage WinGet (coming in separate PR).

File: azure_jumpstart_arcbox/artifacts/Bootstrap.ps1 (hunk 2 above)
Comment (janegilring) [reply azure_jumpstart_arcbox/artifacts/Bootstrap.ps1]: Good idea, I have now added a variable referencing the executable.

**Pull request merged**